includes/utils.inc

```diff
@@ -504,21 +504,21 @@ function wf_crm_get_fields($var = 'fields') {
     $components = $config->enableComponents;
 
     $sets = array(
-      'contact' => array('entity_type' => 'contact', 'label' => t('Contact Fields')),
+      'contact' => array('entity_type' => 'contact', 'label' => t('Contact Fields'), 'custom_fields' => 'separate'),
       'other' => array('entity_type' => 'contact', 'label' => t('Tags and Groups'), 'max_instances' => 1),
-      'address' => array('entity_type' => 'contact', 'label' => t('Address'), 'max_instances' => 9),
-      'phone' => array('entity_type' => 'contact', 'label' => t('Phone'), 'max_instances' => 9),
+      'address' => array('entity_type' => 'contact', 'label' => t('Address'), 'max_instances' => 9, 'custom_fields' => 'combined'),
+      'phone' => array('entity_type' => 'contact', 'label' => t('Phone'), 'max_instances' => 9, 'custom_fields' => 'combined'),
       'email' => array('entity_type' => 'contact', 'label' => t('Email'), 'max_instances' => 9),
       'website' => array('entity_type' => 'contact', 'label' => t('Website'), 'max_instances' => 9),
       'im' => array('entity_type' => 'contact', 'label' => t('Instant Message'), 'max_instances' => 9),
```

Comment:
The above 3 should also be 'combined'

Reply:
ok. Done (I'll squash commits once we've completed review)